code/game/machinery/autolathe.dm

```diff
@@ -174,8 +174,8 @@
 				return
 
 			var/multiplier = text2num(params["multiplier"])
```

Comment:
This shouldn't be text2num at all, if it wasn't then nan would be impossible to send since nan is not a valid JSON number.

Reply:
I'm wrong